src/main/resources/log4j2.properties

```diff
@@ -0,0 +1,15 @@
+appenders = console
+
+appender.console.type = Console
+appender.console.name = STDOUT
+
+appender.console.layout.type = PatternLayout
+appender.console.layout.pattern = %d{HH:mm:ss} %t [$${folio:requestid:-}] [$${folio:tenantid:-}] [$${folio:userid:-}] [$${folio:moduleid:-}] %-5p %-20.20C{1} %m%n
+
+logger.foliospring = debug, STDOUT
```

Comment:
This part was needed for debug only as well

Reply:
To complete logging setup I propose to use approach from documentation, so we need to have 2 files: with pattern layout by default and json layout, you can get examples from this folder: https://github.com/folio-org/folio-spring-support/tree/master/folio-spring-base/src/main/resources

Reply:
regarding active thread %t - we can remain of course in our configuration, there is ticket for folio-spring-base, but it is not yet resolved: https://issues.folio.org/browse/FOLSPRINGB-98